portal-frontend/src/app/features/applications/application-details/application-details.component.ts

```diff
@@ -126,7 +126,10 @@ export class ApplicationDetailsComponent implements OnInit, OnDestroy {
 
       const isSelfApplicant = owners.length === 1 && this.primaryContact?.type.code === OWNER_TYPE.INDIVIDUAL;
       const isGovernmentContact = this.primaryContact?.type.code === OWNER_TYPE.GOVERNMENT;
-      this.needsAuthorizationLetter = !isGovernmentContact && !isSelfApplicant;
+      this.needsAuthorizationLetter = !isSelfApplicant;
```

Comment:
Consider replacing these 4 lines with the following:
```js
this.needsAuthorizationLetter = isGovernmentContact || !isSelfApplicant;
```
It is equivalent, but cleaner and easier to understand.

Reply:
gotcha, made changes